Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.H

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 Neil Zaim
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef BINARY_COLLISION_UTILS_H_
+#define BINARY_COLLISION_UTILS_H_
+
+#include <string>
+
+#include "Particles/MultiParticleContainer.H"
+
+enum struct CollisionType { ProtonBoronFusion, Undefined };
+
+enum struct NuclearFusionType { ProtonBoron, Undefined };
+
+namespace BinaryCollisionUtils{
+
+    NuclearFusionType get_nuclear_fusion_type (const std::string collision_name,
+                                            MultiParticleContainer* const mypc);
+
+    CollisionType get_collision_type (const std::string collision_name,
+                                    MultiParticleContainer* const mypc);
```

Comment:
Since we only call a `const` function on the `mypc`, we can even write:
```suggestion
    NuclearFusionType get_nuclear_fusion_type (const std::string collision_name,
                                               MultiParticleContainer const * const mypc);

    CollisionType get_collision_type (const std::string collision_name,
                                      MultiParticleContainer const * const mypc);
```